pytorch_lightning/utilities/xla_device_utils.py

```diff
@@ -18,11 +18,10 @@
 
 import torch
 
-TORCHXLA_AVAILABLE = importlib.util.find_spec("torch_xla") is not None
-if TORCHXLA_AVAILABLE:
+XLA_AVAILABLE = importlib.util.find_spec("torch_xla") is not None
+
+if XLA_AVAILABLE:
     import torch_xla.core.xla_model as xm
-else:
-    xm = None
```

Comment:
xm is required down below. else it will raise ``xm` is not defined`.

Reply:
good catch :]